Assets/_main-section-desserts.scss

```diff
@@ -0,0 +1,102 @@
+.main-section-desserts {
+  background-color: #fff;
+}
+
+.main-section-desserts-content {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 2.2rem 0 3rem 0;
+}
+
+.main-section-desserts-title {
+  text-align: center;
+}
+
+.main-section-desserts-card-image {
+  max-height: 5rem;
+}
```

Comment:
- [ ] There is the same alignment of an image on the left and text on the right for each individual article. in the middle row, alignments are not similar to the upper and lower row of the article. Please try to fix this for desktop and mobile version.

![image](https://user-images.githubusercontent.com/53019916/125175596-f097a900-e1ea-11eb-9776-3701e20e30c1.png)

Reply:
I fixed it